### PR TITLE
session: speed up test by reduce pre-write back-off time

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
* speed up `TestOptimisticConflicts` test by reduce pre-write back-off time.

before
```
PASS: pessimistic_test.go:349: testPessimisticSuite.TestOptimisticConflicts	43.096s
```

This PR

```
PASS: pessimistic_test.go:349: testPessimisticSuite.TestOptimisticConflicts	2.366s
```

### Check List 

Tests 

 - No code

Code changes

 - Has exported variable/fields change

Side effects

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: session/pessimistic_test.go [hunk 1]
```diff
@@ -60,9 +60,11 @@ func (s *testPessimisticSuite) SetUpSuite(c *C) {
 	s.dom, err = session.BootstrapSession(s.store)
 	s.dom.GetGlobalVarsCache().Disable()
 	c.Assert(err, IsNil)
+	tikv.PrewriteMaxBackoff = 500
```
Comment (jackysp) [inline session/pessimistic_test.go]: Does this lead to data race?

Comment (crazycs520) [reply session/pessimistic_test.go]: Yes, I was hoping for luck  😂 
And I put this change in SetUpSuite to reduce the data race probability.

File: session/pessimistic_test.go [hunk 2]
```diff
@@ -370,7 +372,6 @@ func (s *testPessimisticSuite) TestOptimisticConflicts(c *C) {
 	tk.MustExec("update conflict set c = 4 where id = 1")
 	tk2.MustExec("begin optimistic")
 	tk2.MustExec("update conflict set c = 5 where id = 1")
-	// TODO: ResolveLock block until timeout, takes about 40s, makes CI slow!
```
Comment (tiancaiamao) [inline session/pessimistic_test.go]: I'd rather delete the following line (skip the test), what do you think? @coocood

**Pull request merged**